app/models/form.rb

```diff
@@ -154,48 +154,80 @@ def invalidated_pages(log, current_user = nil)
     pages.reject { |p| p.routed_to?(log, current_user) }
   end
 
-  def invalidated_questions(log)
-    invalidated_page_questions(log) + invalidated_conditional_questions(log)
-  end
```

Comment:
sense check on this, but it did not appear to be being called anywhere in the codebase. very slight risk that the method is being called through some metaprogramming whatnot?